include/linker/ramfunc.ld

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Does it make sense to have, now, an external linker file?
Can we do as for the NoCache PR, and overwrite the main ARM linker script?

Reply:
I don't know if the other platform need this, if so leave this as a external linker file is OK. If not, we'd better do that such as Nocache PR.

Reply:
I think the same applies to the no-cache section.
I believe it is OK to have this linker script code _inlined_ into the main arm linker script (in the very same place).

Reply:
Then, the linker script reader can actually see all sections in-place :)